training/cogvideox_image_to_video_lora.py

```diff
@@ -876,15 +885,10 @@ def load_model_hook(models, input_dir):
                     run_validation(args, accelerator, transformer, scheduler, model_config, weight_dtype)
 
             last_lr = lr_scheduler.get_last_lr()[0] if lr_scheduler is not None else args.learning_rate
-            logs = {"loss": loss.detach().item(), "lr": last_lr}
-            # gradnorm + deepspeed: https://github.com/microsoft/DeepSpeed/issues/4555
-            if accelerator.sync_gradients and accelerator.distributed_type != DistributedType.DEEPSPEED:
-                logs.update(
-                    {
-                        "gradient_norm_before_clip": gradient_norm_before_clip,
-                        "gradient_norm_after_clip": gradient_norm_after_clip,
-                    }
-                )
+            logs.update({
+                "loss": loss.detach().item(),
+                "lr": last_lr,
+            })
```

Comment:
```suggestion
            logs.update({"loss": loss.detach().item(), "lr": last_lr})
```

LoC 119 limit should be fine with it.